docker/ds/bootstrap/setup-ds.sh

```diff
@@ -129,6 +140,12 @@ cp ../../example-v1.json ./config/rest2ldap/endpoints/api
 # From util.sh. Consider moving the logic here...
 configure
 
+
+echo "Putting IDM schema extensions in place"
+cp /var/tmp/schema/* ./db/schema
+
+/var/tmp/bootstrap/setup-idm.sh
+
 ./bin/start-ds
 
 # We only import the ldif on server 1 since we are going to initialize replication from it anywayss.
```

Comment:
s/since we are going to initialize replication from it anywayss/since replication is initiated from server 1/